mm-go-irckit/mmuser.go

```diff
@@ -237,11 +237,13 @@ func (u *User) addUserToChannelWorker(channels <-chan *model.Channel, throttle <
 			for _, post := range strings.Split(p.Message, "\n") {
 				if user, ok := u.mc.Users[p.UserId]; ok {
 					date := ts.Format("2006-01-02")
+					if ((time.Now()-ts)/int64(time.Millisecond)) < 86400 {
```

Comment:
This seems a bit arbitrary - would it be possible to make this configurable?

Reply:
That would be great.

Reply:
This should be configurable for sure.

If this fixes linked issue (#245) I am pretty sure that it's actually more like side effect than proper fix.